tools/utils.py

```diff
@@ -85,7 +85,12 @@ def GuessArchitecture():
     return 'ia32'
   elif id == 'amd64':
     return 'ia32'
+  elif id.startswith('ppc'):
+    return 'ppc'
   else:
+    id = platform.processor()
+    if id == 'powerpc':
+      return 'ppc'
```

Comment:
On different machines in our lab:

PPC LE ubuntu -> ppc64le
PPC BE RHEL65 -> ppc64
AIX  -> powerpc

So that case seems to be for AIX on PPC